test/protocol/protocol.test.js

```diff
@@ -300,7 +305,13 @@ describe('Protocol tests', () => {
     before('Compile smart contracts source', async function compile() {
         this.timeout(20000);
 
+        let accountBalance;
         accounts = await web3.eth.getAccounts();
+        for (let i = 0; i < 10; i += 1) {
+            // eslint-disable-next-line no-await-in-loop
+            accountBalance = await web3.eth.getBalance(accounts[i]);
+            console.debug(`${accounts[i]} with wei balance: ${accountBalance} is available`);
```

Comment:
If you really want to make sure it has correct balance use 'expect' to make sure it does.